front-end/src/app/components/timer/timer.component.ts

```diff
@@ -27,7 +27,7 @@ export class TimerComponent implements OnInit {
     const device = this.app.deviceList.getDevice("front-end");
     if (device !== null) {
       const status = device.status;
-      if (status.get("start") > 0 && status.get("stop") === 0) {
+      if (status.get("gameState") == "in spel") {
```

Comment:
If you hard code this state, point out in the manual that this state is required, or make this state configurable, or perhaps even better, do not check what the status is but what the state of the general timer is.